lib/store/inventory.ex

```diff
@@ -163,6 +178,13 @@ defmodule Store.Inventory do
     |> Repo.insert()
   end
 
+  @doc ""
```

Comment:
It would be nice to have a nice documentation. Can you handle that? 🙏🏼
You can follow all the other files (or even projects) examples.

**Same for all the other functions**